boards/nucleo-l073/include/periph_conf.h

```diff
@@ -95,10 +95,21 @@ static const uart_conf_t uart_config[] = {
         .bus        = APB1,
         .irqn       = USART4_5_IRQn
     },
+    {
```

Comment:
I would prefer to move this as the second UART and not third UART. PA9 and PA10 are generally used this way in other Nucleo 64 boards.